### ISSUE TITLE
SystemError: unknown opcode

### ISSUE THREAD

**Issue by joonis** [dashingsoft/pyarmor#1024]
Running this obfuscated script:

```
def foo(name=None):
    
    def bar(x):
        if not x:
            return 1

    bar(name)
```

Results in the following error:

```
Python 3.8.10 (default, Mar 13 2023, 10:26:41) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import mypackage
>>> mypackage.foo()
XXX lineno: 4, opcode: 0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<mypackage.__init__>", line 7, in foo
  File "<mypackage.__init__>", line 4, in bar
SystemError: unknown opcode
```

**Comment by jondy** [comment]
Found the problem, it's only Python 3.8 issue, it will be fixed in next release.

**Comment by jondy** [comment]
Fixed in 8.0.20
